### PR TITLE
Stress tests: Fix metrics-client logging

### PR DESCRIPTION
**Pull request by sreecha**
The stress test framework programmatically parses the log output of `metrics_client` to get the qps numbers and uploads them to big query. 

Due to some recent changes in grpc core, we are seeing some info log messages being logged to stdout that is overshadowing the logs by metrics_client causing the stress test framework to fail to get the qps.

I made a change to block all logs from grpc library and am using `std::cout` in the metrics_client to log its output.  (I considered using gpr_set_log_verbosity to GPR_ERROR but that seemed like a temporary fix).

### CODE REVIEW

**Comment by sreecha** [comment]
btw, all the stress tests that use the `metrics_client` are currently failing and this PR would fix that.

**Comment by sreecha** [comment]
The test failures are unrelated.

**Comment by vjpai** [comment]
lgtm.

**Comment by vjpai** [comment]
Please rebuild basic and portability tests.

**Comment by vjpai** [comment]
Wait, undo lgtm.

File: test/cpp/interop/metrics_client.cc [hunk 1]
```diff
@@ -76,21 +79,21 @@ bool PrintMetrics(std::unique_ptr<MetricsService::Stub> stub, bool total_only,
...
-        gpr_log(GPR_INFO, "%s: %lld", gauge_response.name().c_str(),
-                gauge_response.long_value());
+        std::cout << gauge_response.name() << ": "
+                  << gauge_response.long_value() << std::endl;
       }
       overall_qps += gauge_response.long_value();
     } else {
-      gpr_log(GPR_INFO, "Gauge %s is not a long value",
-              gauge_response.name().c_str());
+      std::cout << "Gauge %s is not a long value" << gauge_response.name()
```
Comment (vjpai) [inline test/cpp/interop/metrics_client.cc]: Please fix formatting in line 87.

**Comment by vjpai** [comment]
lgtm except for my comment that needs to be resolved.

**Comment by sreecha** [comment]
Thanks @vjpai . I fixed the error

**Comment by vjpai** [comment]
Code LGTM. Please see why tests are failing.

**Comment by sreecha** [comment]
I re-ran the tests and the interop tests passed. The portability tests are still failing but the failures do not seem related. I kicked a rebuild again to see whats up.

The failures in basic tests are known asan failures https://github.com/grpc/grpc/issues/7143 and https://github.com/grpc/grpc/issues/7142.

**Comment by sreecha** [comment]
I merged with latest master and kicked the tests again

**Comment by sreecha** [comment]
The test failures are unlreated (and due to known issues: #7202,  #7142 and #7143)
This can be merged.

**Comment by sreecha** [comment]
@nicolasnoble - can you merge this please?

**Comment by nicolasnoble** [comment]
test this please

**Comment by nicolasnoble** [comment]
Woot! Jenkins seems to listen to commands again.

**Comment by sreecha** [comment]
The failures are unrelated.

**Pull request merged**